keyboards/planck/keymaps/samuel/keymap.c

```diff
@@ -0,0 +1,298 @@
+#include "planck.h"
```

Comment:
Could you replace this:
```suggestion
#include QMK_KEYBOARD_H
```